client/utils/canvas.js

```diff
@@ -158,17 +160,41 @@ export default class Canvas extends React.Component {
     this.painting = false
   }
   render() {
+    this.isChatHidden = this.props.isChatHidden
+    const SecondWrapper = styled.div`
```

Comment:
It's best to move these styled component definitions outside of your render methods.